desktop/libs/notebook/src/notebook/connectors/spark_shell_tests.py

```diff
@@ -133,6 +153,88 @@ def test_create_session_plain(self):
       assert_true(files_properties, session['properties'])
       assert_equal(files_properties[0]['value'], [], session['properties'])
 
+      if SESSIONS.get(session_key):
+        del SESSIONS[session_key]
+
+
+  def test_execute(self):
+    with patch('notebook.connectors.spark_shell._get_snippet_session') as _get_snippet_session:
+      with patch('notebook.connectors.spark_shell.get_spark_api') as get_spark_api:
+        notebook = Mock()
+        snippet = {'statement': 'select * from some_table'}
```

Comment:
can we use a good name in the place of "some_table" for eg. foo, test_table, test etc.

Reply:
Changed